src/reactapp/src/pages/EvaluationForm.js

```diff
@@ -97,8 +73,8 @@ export default function EvaluationForm() {
   const processRatingLevelOptions = async (ratingLevels) => {
     const processedRatingLevels = [{ "label": "N/A - Not Applicable", "value": -1 }, ...ratingLevels.map((level) => {
```

Comment:
@DavidJGapCR The changes are good. A minor request based on a demo I did for the tool today. Can you change 'N/A - Not Applicable' to 'N/A - Not Assessed'. Thanks!

Reply:
Done.